roles/servicetelemetry/defaults/main.yml

```diff
@@ -44,6 +44,23 @@ servicetelemetry_defaults:
             storage_class: ""
             storage_selector: {}
             pvc_storage_request: 20Gi
+    logs:
+      loki:
+        enabled: false
+        node_count: 1
```

Comment:
I'm not finding this used anywhere - is this supposed to be `replication_factor`?

Reply:
yes

Reply:
I changed it to replication_factor